File: Service/ClaimCare/RMA.Service.ClaimCare.Contracts/Entities/VopdDash.cs
using RMA.Common.Entities;

using System.Collections.Generic;

namespace RMA.Service.ClaimCare.Contracts.Entities
{
    public class VopdDash : AuditDetails
    {
        public List<VopdOverview> VopdOverviews { get; set; }
        public int? ProcessingCount { get; set; }
        public int? ProcessedCount { get; set; }
        public int? SuccessfulCount { get; set; }
        public int? UnsuccessfulCount { get; set; }
        public int? Submitted { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/claimcare/src/app/claim-manager/shared/entities/vopd-dash.ts
import { VopdOverview } from './vopd-overview';

export class VopdDash {
  vopdOverviews: VopdOverview[];
  successfulCount: number;
  unsuccessfulCount: number;
  processingCount: number;
  processedCount: number;
  submitted: number;
}
